nvtabular/inference/triton/model_config_util.py

```diff
@@ -0,0 +1,24 @@
+from google.protobuf.descriptor_pool import Default
+from google.protobuf.message_factory import MessageFactory
+
+def_pool = Default()
+msg_factory = MessageFactory(pool=def_pool)
+file_descriptor = def_pool.FindFileByName("model_config.proto")
+if file_descriptor:
```

Comment:
What happens if this isn't found? (I'm not sure we need an else clause, I just don't understand how it works in the file doesn't exist.)

Reply:
so your right by the time you hit this code the protoc will have compiled the code (during install) and then we will always have all the pieces guaranteed. Otherwise the install did not occur correctly.